test/utils/test_pulse_scaling.py

```diff
@@ -138,3 +140,12 @@ def test_forced_rzz_template_match(self):
                 Operator(scale_qc_match.bind_parameters({theta: theta_set}))
             )
         )
+
+    def test_angle_reduction(self):
+        """Test Angle Reduction"""
+        qc = QuantumCircuit(2)
+        qc.rzx(9 * np.pi / 2, 0, 1)
+        pm = PassManager(ReduceAngles(["rzx"]))
+        qc_s = pm.run(qc)
+
+        self.assertAlmostEqual(qc_s.data[0].operation.params[0], np.pi / 2)
```

Comment:
One last thing: please test a few more values, including ones that should reduce to negative values, and edge cases like `-pi`

Reply:
Added a couple more checks.